### PR TITLE
Do not fail the build if ci-logs is not healthy

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Avoid situations like https://github.com/ClickHouse/ClickHouse/actions/runs/8064238811/job/22029154629?pr=60444#step:9:127

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit f49882ee17cccadd05e87970172bc5a0b1f7cb66 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60445/f49882ee17cccadd05e87970172bc5a0b1f7cb66/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/60445/f49882ee17cccadd05e87970172bc5a0b1f7cb66/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Review by maxknv — APPROVED** [review]

File: tests/ci/ci.py [hunk 1]
```diff
@@ -11,17 +8,21 @@
...
+from dataclasses import asdict, dataclass
+from enum import Enum
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Sequence, Set, Union
 
 import docker_images_helper
 import upload_result_helper
 from build_check import get_release_or_pr
-from ci_config import CI_CONFIG, Build, Labels, JobNames
+from ci_config import CI_CONFIG, Build, JobNames, Labels
```
Comment (maxknv) [inline tests/ci/ci.py:20]: and this?

**Review by maxknv — APPROVED** [review]

File: tests/ci/ci.py (hunk 1 above)
Comment (Felixoid) [reply tests/ci/ci.py:20]: It's auto fix by `isort`

**Comment by Felixoid** [comment]
It worked, let's fix the builds!

**Pull request merged**